src/deploy/security/security.tf

```diff
@@ -0,0 +1,30 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+# creates rule in default security group
+resource "aws_security_group_rule" "allow_ssh" {
+  type            = "ingress"
+  from_port       = 22
+  to_port         = 22
+  protocol        = "tcp"
+  cidr_blocks = ["205.206.87.17/32"]
+  security_group_id = "sg-4c6cea0a"
+}
+
+# creates rule in default security group
+resource "aws_security_group_rule" "allow_http" {
+  type            = "ingress"
+  from_port       = 80
+  to_port         = 8282
```

Comment:
The API is running on 8282, the front end dev server on 8080 and the production server on 5000